src/text_summarizer.py

```diff
@@ -0,0 +1,47 @@
+import spacy
+
+nlp = spacy.load("en_core_web_lg")
+
+def get_summary(text, num_sentences=3):
```

Comment:
I think we can infer the number of sentences directly from the text using the `split` built-in function, perhaps something like "Hello".split('.'). The only problem that comes to my mind with this approach is dealing with ellipses and such. What do you think?